frankly034/Dantown
# Add card rate and card resources

#### What does this PR do?
Setup email verification feature 

#### Description of Task to be completed?
- add card and cardrate:
-- models
-- controllers
-- validators
-- routes
--- GET, POST, PUT, PATCH, DELETE /card-rates
--- GET /cards
-- migrations

#### How should this be manually tested?
* clone the repo
* run `npm install`
* start the app with `npm run start:dev`
* visit POST, GET, PATCH, DELETE /card-rates
* visit GET /cards
* You should get a `200` status on success or `400` or `404` on failure

#### Any background context you want to provide?
* must have `node` installed

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
